Misc/NEWS.d/next/Library/2019-04-08-09-24-36.bpo-31904.ab03ea.rst

```diff
@@ -0,0 +1 @@
+Skip setting RLIMIT_FSIZE and RLIMIT_CPU test case for VxWorks
```

Comment:
Please move this NEWS entry to the Misc/NEWS.d/next/Tests/ directory, and mention the name of the fixed test, for example:

```
Port test_resource to VxWorks: skip tests cases setting RLIMIT_FSIZE and RLIMIT_CPU.
```